searx/engines/tavily.py

```diff
@@ -6,18 +6,18 @@
    Before reporting an issue with this engine,
    please consult `API error codes`_.
 
-Tavily_ search API (AI engine).  This engine implements the REST API
+Tavily_ search API (AI engine). This engine implements the REST API
```

Comment:
The double space after the dot of sentence end, its intended.

Typographic tools use the two spaces after the end of a sentence to distinguish from the period of an abbreviation (by instance some editors adjust their automated line breaks accordingly).